task/cerberus-check/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- startxfr
+reviewers:
+- startxfr
```

Comment:
@startxfr you have added the OWNERS file, are you willing to maintain these tasks in future? If so, can you please add yourself into the org? Probably you can refer the PR https://github.com/tektoncd/community/pull/680

Reply:
Done in the PR https://github.com/tektoncd/community/pull/734